readme.md

```diff
@@ -1,12 +1,12 @@
 # Mocha generator [![Build Status](https://secure.travis-ci.org/yeoman/generator-mocha.svg?branch=master)](http://travis-ci.org/yeoman/generator-mocha)
 
-> Yeoman generator for [Mocha](http://visionmedia.github.io/mocha/)
+> Yeoman generator for [Mocha](http://mochajs.org/)
 
 
 ## Install
 
 ```
-$ npm install --global generator-mocha
+$ npm install -g generator-mocha
```

Comment:
None strong enough to truly warrant changing it back to the short from. I copy-pasted the command and got an error, but I had overzealously copied the `$` too. 
I'll remove this change from my PR.